components/konflux-rbac/policies/bootstrap-tenant-namespace/bootstrap-tenant-namespace-clusterpolicy.yaml

```diff
@@ -34,3 +34,151 @@ spec:
           kind: ServiceAccount
           namespace: '{{request.object.metadata.name}}'
           name: appstudio-pipeline
+  - name: create-allow-from-console-namespaces-networkpolicy
+    match:
+      any:
+      - resources:
+          kinds:
+          - Namespace
+          selector:
+            matchLabels:
+              konflux-ci.dev/type: tenant
+    generate:
+      generateExisting: true
```

Comment:
what does `generateExisting` mean? (I didn't find it in the documentation)

Reply:
this way we'll create the resources in all the already existing tenant namespaces that don't have them. Nothing will happen if they already exist. After some time, when we are sure every namespace has been fixed, we can change this value to `false`